src/core/gltf2importer/gltf2parser.cpp

```diff
@@ -485,6 +519,85 @@ const GLTF2Context *GLTF2Parser::context() const
     return m_context;
 }
 
+void GLTF2Parser::generateMissingTangents(const QString &basePath, QJsonObject &rootObject)
```

Comment:
I think that this should be refactored to go in the exporter part - maybe by storing a flag in one of the GLTF2Context struct which indicates which mesh had its tangents computed and then by adding a pass here before the draco compression (which might compress the tangent attribute) : https://github.com/KDAB/kuesa/blob/dev/src/core/gltf2exporter/gltf2exporter_p.cpp#L199 ?